gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobScheduler.java

```diff
@@ -116,7 +119,8 @@ public GobblinHelixJobScheduler(Properties properties,
     this.jobCatalog = jobCatalog;
     this.metricContext = Instrumented.getMetricContext(new org.apache.gobblin.configuration.State(properties), this.getClass());
 
-    int metricsWindowSizeInMin = ConfigUtils.getInt(ConfigUtils.propertiesToConfig(this.properties),
+    Config jobConfig = ConfigUtils.propertiesToConfig(this.properties);
```

Comment:
Conversion of all the properties to Config looks unnecessary to me.

Reply:
This should be the recommended pattern, so we can use ConfigUtils for the rest of the config gets, without incurring repeated txformations between properties and Config.